### PR TITLE
Remove unused parts of the OpenCV build

### PR DESCRIPTION
**Pull request by Gold856**
Aruco module has been replaced by objdetect, unused date variables were removed, and manifest blocks that didn't do anything were removed from zip tasks.

### CODE REVIEW

**Comment by PeterJohnson** [comment]
Is aruco not needed by wpical anymore?  It’s why we added it.

**Comment by Gold856** [comment]
We still need aruco, but the aruco module has been obsoleted by the objdetect library (has been for a little while), so we don't need the external module. I also tested building wpical with a locally published build of opencv without the aruco module or aruco module headers (just the objdetect headers) and it works, so this should be fine.

File: publish.gradle [hunk 1]
```diff
@@ -215,19 +198,6 @@ task cppHeadersZip(type: Zip, dependsOn: make) {
...
-                    "Implementation-Title": "OpenCV Native Libraries, ${project.ext.platformClassifier}",
-                    "Implementation-Version": pubVersion,
-                    "Implementation-Vendor": "Itseez",
-
-                    "Bundle-Name": "${archiveBaseName.get()}",
-                    "Bundle-Version": pubVersion,
-                    "Bundle-License": "https://opensource.org/licenses/BSD-3-Clause",
-                    "Bundle-Vendor": "WPILib")
-        }
-
```
Comment (PeterJohnson) [inline publish.gradle:230]: What’s the reason to remove these?

File: build.gradle [hunk 2]
```diff
@@ -63,9 +60,11 @@ ext {
             '-DCMAKE_SKIP_RPATH=FALSE',
             '-DCMAKE_BUILD_RPATH_USE_ORIGIN=TRUE',
             '-DBUILD_opencv_apps=OFF',
+            '-DBUILD_opencv_js_bindings_generator=OFF',
+            '-DBUILD_opencv_python_bindings_generator=OFF',
+            '-DBUILD_opencv_python_tests=OFF',
```
Comment (PeterJohnson) [inline build.gradle]: Why the changes to Python?

File: publish.gradle (hunk 1 above)
Comment (Gold856) [reply publish.gradle:230]: The manifest blocks are in the middle of the header and source zip tasks, which means they do nothing. There's no manifest in either of the resulting archives.

File: build.gradle (hunk 2 above)
Comment (Gold856) [reply build.gradle]: I thought when I looked at the Mac builds, they included Python code that wasn't supposed to be there. Seems like that isn't the case, since the runners don't even have the necessary dependencies to build Python libraries. Change has been reverted since it's unnecessary.

**Review by PeterJohnson — APPROVED** [review]
